index.html

```diff
@@ -30,22 +40,22 @@ <h1 class="name" >Emilia Zambrano</h1>
     <section class="projects">
         <h1 class="h1projects">Projects</h1>
 
-        <article class="work">
-            <img class="imgArticle" src="./images/proj_1.png" alt="First Project">
-            <article class="info">
-                <h2 class="proj-name">Project name goes here</h2>
-                <ul class="box">
+        <article class="work work1">
```

Comment:
- [ ] Below images show what your page looks like on medium screens which does not make it responsive. Please ensure that the page is responsive across different screen sizes.

![image](https://user-images.githubusercontent.com/43995830/160063471-fd18454a-d655-4468-acbd-fa73a5b9f08a.png)

![image](https://user-images.githubusercontent.com/43995830/160063516-90ade23a-0ffb-4f5e-8c53-68425babbe4a.png)